bindings/python/iota_sdk/client/_high_level_api.py

```diff
@@ -82,40 +81,36 @@ def _call_method(self, name, data=None):
         no payload.
         """
 
-    # TODO: this should return `List[OutputResponse]`, not `List[OutputWithMetadata]`
-    # https://github.com/iotaledger/iota-sdk/issues/1921
     def get_outputs(
-            self, output_ids: List[OutputId]) -> List[OutputWithMetadata]:
-        """Fetch OutputWithMetadata from provided OutputIds (requests are sent in parallel).
+            self, output_ids: List[OutputId]) -> List[OutputResponse]:
+        """Fetch OutputResponse from provided OutputIds (requests are sent in parallel).
 
         Args:
             output_ids: A list of output ids.
 
         Returns:
-            A list of corresponding `OutputWithMetadata` objects.
+            A list of corresponding `OutputResponse` objects.
         """
         outputs = self._call_method('getOutputs', {
             'outputIds': list(map(lambda o: o.output_id, output_ids))
         })
-        return [OutputWithMetadata.from_dict(o) for o in outputs]
+        return [OutputResponse.from_dict(o) for o in outputs]
 
-    # TODO: this should return `List[OutputResponse]`, not `List[OutputWithMetadata]`
-    # https://github.com/iotaledger/iota-sdk/issues/1921
     def get_outputs_ignore_errors(
```

Comment:
This is called `get_outputs_ignore_not_found` in Rust

Reply:
hmm, just realized this is also true for core and nodejs